### PR TITLE
Github pages added

### PR DESCRIPTION
**Pull request by DamonU2**
- Moved github pages files from data repo to docs folder
- Action to generate release assets when release is created
- Preview maps updated

Once this is merged github pages will need to be enabled using the docs folder

### CODE REVIEW

**Comment by jvanulde** [comment]
@DamonU2 let's set up a meeting with @tieganh to review. This needs some explanation.

**Comment by jvanulde** [comment]
@tieganh we are moving the gh-pages from the data repo here. The reason being that it allows us finer grained control over what is released and where. It also keeps the scripts that process/access the outputs in this repo. It's much easier to manage than having relevant resources spread across multiple repositories.

**Review by jvanulde — CHANGES_REQUESTED** [review]
I didn't look at everything but I reviewed the key files. Just one question for you.

File: docs/_data/dsra.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ scenarios:
   name: IDM7p1_Sidney
   title: Sidney - Magnitude 7.1
 - description:
-    en: .
+    en: Magnitude 9.0 scenario based on complete rupture of the CanSHM6 Hazard Model implementation of the Cascadia Interface Best Fault.
```
[the remark this answers is not in the document]
Comment (DamonU2) [reply docs/_data/dsra.yml:21]: Huh. Not sure how that happened.

[the remark this answers is not in the document]
Comment (DamonU2) [reply docs/_data/dsra.yml:21]: It's just in a weird order. The script to generate that put them alphabetically, so the description is the first thing.

**Pull request merged**